docs/sync.js

```diff
@@ -170,6 +173,7 @@ class User {
      * @param {string} adminToken - existing admin token
      * @param {string} [server] - authentication server
      * @return {User} - admin user populated with the given token and server
+     * @returns {Promise} returns a Promise if the callback was not supplied
```

Comment:
`Realm.Sync.User.adminUser` is synchronous and doesn't return a Promise, nor take a callback.